### DISCUSSION TITLE
How do I install darkreader

### DISCUSSION THREAD

**Discussion by ModProg** [Jelmerro/Vieb#298] [Q&A] [answered]
I tried searching for it. but didn't find anything really.

**Comment by Jelmerro** [comment]
You can use `:extensions install` when on an extension page in either the Chrome or Firefox store. See `:h extensions` for details.

**Comment by erwin** [comment]
Apparently it's no possible to enable / disable or interact with installed extensions for now...

As a workaround, I added a `:dark` and `:light` command to my `viebrc` that can enable and disable "dark mode"... It's not 100% prefect, but it seems to be to be a working solution for now.

```
command! light <:extensions remove 3864423> <Bar> <:colorscheme light>

command! dark ba <:open https://addons.mozilla.org/en-US/firefox/addon/darkreader/> <Bar> <:extension install> <Bar> <:colorscheme default>
```

**Comment by Jelmerro** [comment, marked as answer]
Darkreader is now integrated into Vieb, see `:h darkreader` in the new 8.1.0 release!
